<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>目录</string>
     </property>
     <addaction name="action_4"/>
     <addaction name="action_5"/>
     <addaction name="action_6"/>
     <addaction name="actionW"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="menu_2"/>
   </widget>
   <widget class="QMenu" name="menu_W">
    <property name="title">
     <string>窗口(&amp;W)</string>
    </property>
    <addaction name="actionClose"/>
    <addaction name="actionCloseAll"/>
    <addaction name="separator"/>
    <addaction name="actionTile"/>
    <addaction name="actionCascade"/>
    <addaction name="separator"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
    <addaction name="separator"/>
    <addaction name="actionWordCard"/>
   </widget>
   <widget class="QMenu" name="menu_L">
    <property name="title">
     <string>标签(&amp;L)</string>
    </property>
    <addaction name="actionAddLab"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>格式</string>
    </property>
    <addaction name="actionTextBold"/>
    <addaction name="actionTextItalic"/>
    <addaction name="actionTextUnderline"/>
    <addaction name="separator"/>
    <addaction name="actionAlignLeft"/>
    <addaction name="actionAlignCenter"/>
    <addaction name="actionAlignRight"/>
    <addaction name="actionAlignJustify"/>
    <addaction name="separator"/>
    <addaction name="actionTextColor"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_W"/>
   <addaction name="menu_L"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionTextBold"/>
   <addaction name="actionTextItalic"/>
   <addaction name="actionTextUnderline"/>
   <addaction name="separator"/>
   <addaction name="actionAlignLeft"/>
   <addaction name="actionAlignCenter"/>
   <addaction name="actionAlignRight"/>
   <addaction name="actionAlignJustify"/>
   <addaction name="separator"/>
   <addaction name="actionSetColor"/>
   <addaction name="actionDefColor"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/new.png</normaloff>:/myImage/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建文件(&amp;N)</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/open.png</normaloff>:/myImage/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件(&amp;O)...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/save.png</normaloff>:/myImage/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/saveAs.png</normaloff>:/myImage/images/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/exit.png</normaloff>:/myImage/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/undo.png</normaloff>:/myImage/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/redo.png</normaloff>:/myImage/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>恢复(&amp;R)</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/cut.png</normaloff>:/myImage/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/copy.png</normaloff>:/myImage/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/paste.png</normaloff>:/myImage/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>关闭(&amp;O)</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="text">
    <string>关闭所有窗口(&amp;A)</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>平铺(&amp;T)</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="text">
    <string>层叠(&amp;C)</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/next.png</normaloff>:/myImage/images/next.png</iconset>
   </property>
   <property name="text">
    <string>下一个(&amp;X)</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/previous.png</normaloff>:/myImage/images/previous.png</iconset>
   </property>
   <property name="text">
    <string>前一个(&amp;V)</string>
   </property>
  </action>
  <action name="action_A">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/about.png</normaloff>:/myImage/images/about.png</iconset>
   </property>
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
  <action name="action_Qt_Q">
   <property name="text">
    <string>关于Qt(&amp;Q)</string>
   </property>
  </action>
  <action name="actionAddLab">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionPython">
   <property name="text">
    <string>python</string>
   </property>
  </action>
  <action name="action66">
   <property name="text">
    <string>66</string>
   </property>
  </action>
  <action name="action756">
   <property name="text">
    <string>756</string>
   </property>
  </action>
  <action name="actionWordCard">
   <property name="text">
    <string>WordCard</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>隐</string>
   </property>
  </action>
  <action name="actionTextBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/textbold.png</normaloff>:/myImage/images/textbold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionTextItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/textitalic.png</normaloff>:/myImage/images/textitalic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionTextUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/textunder.png</normaloff>:/myImage/images/textunder.png</iconset>
   </property>
   <property name="text">
    <string>UnderLine</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionAlignLeft">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/textleft.png</normaloff>:/myImage/images/textleft.png</iconset>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionAlignCenter">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/textcenter.png</normaloff>:/myImage/images/textcenter.png</iconset>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAlignRight">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/textright.png</normaloff>:/myImage/images/textright.png</iconset>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAlignJustify">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/textjustify.png</normaloff>:/myImage/images/textjustify.png</iconset>
   </property>
   <property name="text">
    <string>Justify</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionTextColor">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionSetColor">
   <property name="text">
    <string>setColor</string>
   </property>
  </action>
  <action name="actionDefColor">
   <property name="text">
    <string>DefColor</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>每日记录</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>目标文件</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>背景图</string>
   </property>
  </action>
  <action name="actionW">
   <property name="text">
    <string>Word背景图</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myImage.qrc"/>
 </resources>
 <connections/>
</ui>
